Dockerfile

```diff
@@ -66,30 +81,44 @@ RUN source /app/scripts/set_compiler_env.sh \
     github.com/Kagami/go-face
 
 COPY api /app/api
-RUN source /app/scripts/set_compiler_env.sh \
+RUN export $(cat /env) \
+  && go env \
   && go build -v -o photoview .
 
 ### Build release image ###
-FROM --platform=${BUILDPLATFORM:-linux/amd64} debian:bookworm-slim AS release
+FROM debian:bookworm-slim AS release
 ARG TARGETPLATFORM
 
 # See for details: https://github.com/hadolint/hadolint/wiki/DL4006
 SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
 
 COPY scripts/install_runtime_dependencies.sh /app/scripts/
-RUN chmod +x /app/scripts/install_runtime_dependencies.sh \
+RUN --mount=type=bind,from=api,source=/dependencies/,target=/dependencies/ \
```

Comment:
`COPY --form` creates another layer, while `RUN --mount` won't. Since there is no need to keep the `deb` file after installation, I prefer not adding another layer.